examples/benchmark_buffer_size/benchmark.py

```diff
@@ -90,16 +147,129 @@
     has_gpu = True
 except AssertionError:
     gpu_gb = 0
-    gpu_name = "CPU"
+    gpu_name = cpu_name
     has_gpu = False
 
+
 # Parse redis URI
 redis_uri = args.redis
 redis_host = redis_uri.split("://")[1].split(":")[0]
 redis_port = int(redis_uri.split("://")[1].split(":")[1])
 
 redis_client = redis.Redis(host=redis_host, port=redis_port, db=0)
 
+# Build range of buffer sizes to test
+buffer_sizes = [
+    1 << i for i in range(args.start_buffer_size, args.end_buffer_size)
+]
+plaid_sizes = [
+    1 << i
+    for i in range(args.start_plaid_buffers - 1, args.end_plaid_buffers + 1)
+]
+
+
+def log(
+    total_sz: int,
+    start: float,
+    end: float,
```

Comment:
Individual `time.monotonic()` timepoints are defined as have no meaning—they aren't actual "date" timestamps and can't be converted to absolute dates or reliably compared with other systems—only their differences have meaning, so it would be more helpful to just log `end - start` durations in these